test/schematype.test.js

```diff
@@ -208,6 +208,25 @@ describe('schematype', function() {
     });
   });
 
+  it('merges default validators (gh-14070)', function() {
```

Comment:
i dont understand what this test tests, because it passes even latest master (without the changes)

if i understand the code in `schematype.js` correctly, shouldnt there be options passed to `new TestSchemaType();`?

Reply:
You're right, fixed in 4d3243f. This is additional motivation for this change: the following retains schematype-level default validators, even without the change in this PR

```javascript
const schema = new Schema({ arr: [Number] });
schema.path('arr').validate(v => v.length === 2);
```

The following does not without this change

```javascript
const schema = new Schema({ arr: { type: [Number], validate: v => v.length === 2 } });
```

A bit of a gotcha.

Reply:
> The following does not without this change
> const schema = new Schema({ arr: { type: [Number], validate: v => v.length === 2 } });
> A bit of a gotcha.

with this PR the defaults are also applied there then?